luaui/Widgets/gfx_DrawUnitShape_GL4.lua

```diff
@@ -245,44 +265,49 @@ local function DrawUnitGL4(unitID, unitDefID, px, py, pz, rotationY, alpha, team
 		Spring.Debug.TraceFullEcho(nil,nil,nil,"DrawUnitGL4")
 		return nil
 	end
-
-	local elementID = pushElementInstance(DrawUnitVBOTable, {
-			px, py, pz, rotationY,
-			alpha, 0, teamcoloroverride,highlight ,
-			teamID, 0,
-			0,0,0,0
-		},
-		uniqueID,
+	
+	instanceCache[1], instanceCache[2], instanceCache[3], instanceCache[4] = px, py, pz, rotationY
+	instanceCache[5], instanceCache[6], instanceCache[7], instanceCache[8] = alpha, 0, teamcoloroverride, highlight
+	instanceCache[9] = teamID
+	
+	local elementID = pushElementInstance(
+		DrawUnitVBOTable,
+		instanceCache,
+		updateID,
 		true,
 		nil,
 		unitID,
 		"unitID")
-	return uniqueID
+	return updateID
 end
 
-
-local function DrawUnitShapeGL4(unitDefID, px, py, pz, rotationY, alpha, teamID, teamcoloroverride, highlight)
-	-- Documentation for DrawUnitShapeGL4:
-	--	unitDefID: which unitDef do you want to draw
-	-- px, py, py: where in the world to do you want to draw it
-	-- rotationY: Angle in radians on how much to rotate the unit around Y,
-		-- 0 means it faces south, (+Z),
-		-- pi/2 points west (-X)
-		-- -pi/2 points east
-	-- alpha: the transparency level of the unit
-	-- teamID: which teams teamcolor should this unit get, leave nil if you want to keep the original teamID
-	-- teamcoloroverride: much we should mix the teamcolor into the model color [0-1]
-	-- highlight: how much we should add a highlighting animation to the unit (blends white with [0-1])
-	-- returns: a unique handler ID number that you should store and call StopDrawUnitGL4(uniqueID) with to stop drawing it
-	-- note that widgets are responsible for stopping the drawing of every unit that they submit!
-	uniqueID = uniqueID + 1
-
+---DrawUnitShapeGL4(unitDefID, px, py, pz, rotationY, alpha, teamID, teamcoloroverride, highlight, updateID, ownerID)
```

Comment:
Shouldn't need to list the function name or the parameters here, those will be captured with better formatting by IDE tools